app/assets/stylesheets/application.scss

```diff
@@ -593,22 +586,30 @@ img {
 }
 
 .small_profile_picture_div {
-  width: 100px;
-  height: 100px;
+  width: 1px;
```

Comment:
Why is this 1px?

Reply:
This is a div element made to behave like a table cell. Since we want to control the width by way of the content's width (which in this case is the Avatar component), we can reduce the number of places to update width information from two places to one.